.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"  # See docs, they say not to use '.github'
+    schedule:
+      interval: "weekly"
+    commit-message:
```

Comment:
[question] I wish the docs were a bit more explicit.  

If you don't include a `commit-message` section, like [OTIO](https://github.com/AcademySoftwareFoundation/OpenTimelineIO/blob/main/.github/dependabot.yml), then the list of dependencies updated is in the [commit message](https://github.com/AcademySoftwareFoundation/OpenTimelineIO/commits/main).

However, then the docs say:

> `include: "scope"` specifies that any prefix is followed by a list of the dependencies updated in the commit.

This would seem to imply, then, that if you have a `prefix` but don't add `include: "scope"`, then the commit message won't list the updated dependencies?

That seems daft to me, but, like I say, the docs aren't super clear.

I guess we can try without `include: "scope"` and see if the generated messages are reasonable?

Reply:
The ones it made in my account seem ok?

Reply:
Yup, looks fine. Why can't docs just make sense.